proposal/color-4-new-spaces.md

```diff
@@ -0,0 +1,1645 @@
+      `space` be the value of `$space`.
+
+  * If `$channel` is not the name of a channel in the color-space `space`,
+    throw an error.
+
+  * Return `true` if the channel `$channel` is [powerless](#powerless-components)
+    in `color`, otherwise return `false`.
+
+### `color.is-in-gamut()`
+
+* ```
+  is-in-gamut($color, $space)
+  ```
+
+  * If `$color` is not a color, throw an error.
+
+  * Let `space` be the value of `$space` if specified, or the result of calling
+    `space($color)` otherwise.
+
+  * If `space` is not a valid [color space][], throw an error.
+
+  * Let `gamut` be the [color gamut](#color-gamuts) associated with `space`
+    if an association is defined, or the value of `space` otherwise.
+
+  * Let `color` be the result of calling `to-space($color, space)`.
+
+  * For all bounded channels in `space`, if the associated channel value in
+    `$color` is outside the bounded range, return `false`.
+
+  * Otherwise, return `true`.
+
+### `color.to-gamut()`
+
+* ```
+  to-gamut($color, $space)
+  ```
+
+  * If `$color` is not a color, throw an error.
+
+  * Let `origin-space` be the result of calling `space($color)` otherwise.
+
+  * Let `target-space` be the value of `$space` if specified, or the value of
+    `origin-space` otherwise.
+
+  * If `target-space` is not a valid [color space][], throw an error.
+
+  * Return the result of [gamut mapping][] with `$color` as the
+    origin color, `origin-space` as the origin color space, and
+    `target-space` as the destination color space.
+
+[gamut mapping]: #gamut-mapping
+
+### `color.channel()`
+
+> Note that channel values are stored as specified, even if those values are
+> out-of-gamut for the [color space][] used. Similarly, this color-channel
+> inspection function may return out-of-gamut channel values.
+
+* ```
+  channel($color, $channel, $space)
+  ```
+
+  * If `$space` is null:
+
+    * Let `space` be the result of calling `space($color)`, and let `color` be
+      the value of `$color`.
+
+  * Otherwise:
+
+    * Let `color` be the result of calling `to-space($color, $space)`, and let
+      `space` be the value of `$space`.
+
+  * If `space` is a known [color space][], let `channels` be a map of channel
+    names defined for `space`, and their corresponding values in `color`, or a
+    map with 1-indexed number keys and their corresponding values in `color`
+    otherwise.
+
+  * Let `value` be the result of calling `map.get(channels, $channel)`.
+
+  * If `value` is `null`, throw an error.
+
+  * Otherwise, return `value`.
+
+
+## Modified Color Module Functions
+
+### `color.hwb()`
+
+These functions are now deprecated. Authors should use global `hwb()` instead.
+
+> Channel clamping and scaling have been removed from the global function,
+> since we now allow out-of-gamut color-channels to be stored as specified.
+
+* ```
+  hwb($channels)
+  ```
+
+  This function is available as a global function named `hwb()`.
```

Comment:
The `color.hwb($channels)` function already exists (https://sass-lang.com/documentation/modules/color#hwb) so I wasn't intending to add anything new here - only redirect to the appropriate version while deprecating. Would that be better written as a call to the global, rather than an overload?

```css
  * Return the result of calling the global function `hwb($channels)`.
```

Reply:
Yeah, I think it's a bit confusing to say that only one overload of a function is available as a global—explicitly having a "calls" relationship between them makes more sense to me.